policy/modules/system/getty.te

```diff
@@ -92,6 +96,12 @@ logging_send_syslog_msg(getty_t)
 
 miscfiles_read_localization(getty_t)
 
+allow getty_t sysctl_t:dir search;
+allow getty_t sysctl_kernel_t:dir search;
+allow getty_t sysctl_kernel_t:file { read open };
```

Comment:
not allowed to reference sysctl_t and sysctl_kernel_t here. Use the appropriate existing interfaces